docs/blog/newsletter-september-2022.rst

```diff
@@ -0,0 +1,91 @@
+.. post:: September 06, 2022
+   :tags: newsletter
+
+##########################################
+Write the Docs Newsletter – September 2022
+##########################################
+
+Hey, everyone, the newsletter is back! We took one of our regularly scheduled breaks, but now we're back with regularly exceptional insights from our community.
```

Comment:
Reword double regular?

Reply:
:disappointed: OK, fine. I put it in and took it out a few times. But if it doesn't work for others, I can take it out.

Reply:
Just a suggestion, ignore as needed. <3